src/main/java/appeng/core/sync/network/NetworkHandler.java

```diff
@@ -95,52 +110,9 @@ public void newConnection( final PlayerLoggedInEvent loginEvent )
 		}
 	}
 
-	@SubscribeEvent
-	public void serverPacket( final ServerCustomPacketEvent ev )
-	{
-		final NetHandlerPlayServer srv = (NetHandlerPlayServer) ev.packet.handler();
-		if( this.serveHandler != null )
-		{
-			this.serveHandler.onPacketData( null, ev.packet, srv.playerEntity );
-		}
-	}
-
-	@SubscribeEvent
-	public void clientPacket( final ClientCustomPacketEvent ev )
-	{
-		if( this.clientHandler != null )
-		{
-			this.clientHandler.onPacketData( null, ev.packet, null );
-		}
-	}
-
-	public String getChannel()
-	{
-		return this.myChannelName;
-	}
-
-	public void sendToAll( final AppEngPacket message )
-	{
-		this.ec.sendToAll( message.getProxy() );
-	}
-
-	public void sendTo( final AppEngPacket message, final EntityPlayerMP player )
-	{
-		this.ec.sendTo( message.getProxy(), player );
-	}
-
-	public void sendToAllAround( final AppEngPacket message, final NetworkRegistry.TargetPoint point )
-	{
-		this.ec.sendToAllAround( message.getProxy(), point );
-	}
-
-	public void sendToDimension( final AppEngPacket message, final int dimensionId )
-	{
-		this.ec.sendToDimension( message.getProxy(), dimensionId );
-	}
-
-	public void sendToServer( final AppEngPacket message )
+	private static final <REQ extends AppEngPacket, REPLY extends AppEngPacket> void registerMessage( Class<? extends IMessageHandler<REQ, REPLY>> messageHandler, Class<REQ> requestMessageType, Side side )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Redundant 'final' modifier. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.modifier.RedundantModifierCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00119)